src/app/modules/browse/browse-controller.js

```diff
@@ -140,9 +140,29 @@
             }
             return _localize(message, params);
         }
+        function setHiddenBooksMessages() {
+            if ($scope.allLicenses) {
+                $scope.hiddenBooksMessage = "";
+                $scope.hiddenBooksToggleMessage = _localize("Hide");
+                $scope.hiddenBooksClosingMessage = _localize(" books with restricted/unknown licenses.");
+            } else {
+                var hiddenCount = $scope.numHiddenBooks;
+                if (hiddenCount === 1) {
+                    $scope.hiddenBooksMessage = _localize("Hiding 1 other book because it has a restricted/unknown license. ");
+                } else {
+                    var params = {
+                        count: hiddenCount
+                    };
+                    $scope.hiddenBooksMessage = _localize("Hiding {count} other books because they have restricted/unknown licenses. ", params);
+                }
+                $scope.hiddenBooksToggleMessage = _localize("Show them");
```

Comment:
If you're going to be careful about singular/plural, the '1' case should really make this 'Show it'.